ledger/eval/txntracer.go

```diff
@@ -48,12 +48,28 @@ type StateDeltaSubset struct {
 }
 
 func convertStateDelta(delta ledgercore.StateDelta) StateDeltaSubset {
+	kvmods := make(map[string]ledgercore.KvValueDelta)
+	for k1, v1 := range delta.KvMods {
+		kvmods[k1] = v1
+	}
+	txids := make(map[transactions.Txid]ledgercore.IncludedTransactions)
+	for k2, v2 := range delta.Txids {
+		txids[k2] = v2
+	}
+	txleases := make(map[ledgercore.Txlease]basics.Round)
+	for k3, v3 := range delta.Txleases {
+		txleases[k3] = v3
```

Comment:
Codecov is complaining: Added line #L61 was not covered by tests

Is this possibly covered in a different package?

Reply:
No I don't think it's covered, I'll add a lease to the txns in the tracer test to make sure it's covered.

Reply:
Btw, another example that gets me pining for go 1.18+ generics. We could have one liner copies.